src/Console/Command/Hook.php

```diff
@@ -76,13 +121,23 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         // use ansi coloring only if not disabled in captainhook.json
         $output->setDecorated($config->useAnsiColors());
-        // use the configured verbosity to manage general output verbosity
-        $output->setVerbosity(IOUtil::mapConfigVerbosity($config->getVerbosity()));
+
+        // If the verbose option is present on the command line, then use it.
+        // Otherwise, use the verbosity setting from the configuration.
+        if (!$input->hasOption('verbose') || !$input->getOption('verbose')) {
+            $output->setVerbosity(IOUtil::mapConfigVerbosity($config->getVerbosity()));
+        }
```

Comment:
I did this because I found that `captainhook` didn't respect the verbosity option (i.e., `-v`, `-vv`, and `-vvv`) passed on the command line. It was always setting the output to whatever was configured in `captainhook.json`.

IMO, if you pass verbosity on the command line, it should override whatever is configured in `captainhook.json`.